charts/nexus3/Chart.yaml

```
@@ -23,9 +23,5 @@ maintainers:
 annotations:
   artifacthub.io/alternativeName: nexus
   artifacthub.io/changes: |
-    - kind: added
-      description: "Added `service.additionalPorts.hosts` (list) for multi-hosts support."
     - kind: changed
-      description: "Updated the _Nexus3_ OCI image to [v3.70.1](https://github.com/sonatype/nexus-public/releases/tag/release-3.70.1-02)."
-    - kind: deprecated
-      description: "Deprecated `service.additionalPorts.host` in favour of `service.additionalPorts.hosts`."
+      description: "Updated the _Nexus3_ OCI image to [v3.70.2](https://hub.docker.com/layers/sonatype/nexus3/3.70.2-java17-alpine/images/sha256-2673b4f2ad59838d87ea066b6a275d4520b0aa1c0271f24954276eaa6686adf0?context=explore)."
```

```suggestion
      description: "Updated the _Nexus3_ OCI image to `v3.70.2`."
```
See above (not that it matters here as all markdown is stripped)

---

fixed